scripts/commands/updateApiDocs.ts

```diff
@@ -343,7 +390,7 @@ async function convertHtmlToMarkdown(
     JSON.stringify(toc, null, 2) + "\n",
   );
 
-  if (pkg.hasSeparateReleaseNotes) {
+  if (pkg.hasSeparateReleaseNotes && !historical) {
```

Comment:
Instead, we want to delete the generated release note file and replace with whatever the prior notes were: https://github.com/Qiskit/documentation/issues/330

Might want to refactor the code first: https://github.com/Qiskit/documentation/issues/370

You can punt on this in this PR by removing release notes entirely for now. But that may be more complex because you have to make sure `_toc.json` is correct.

Reply:
I changed the code to continue creating the `index.md`, delete the generated release note files and replace them with a copy of the non-historical version release notes. Also, the `_toc.json` contains the new path to each release note in its subfolder.